e2e/app.e2e.spec.ts

```diff
@@ -32,21 +36,22 @@ describe('iam-cache-server E2E tests', () => {
   }
 
   beforeAll(async () => {
+    await import('../src/scripts/truncate-db');
```

Comment:
It seems that just importing the script has the side-effect of executing the script then? Is this to avoid needing to call a function after the import?

Reply:
Not exactly. I didn't define truncate-db as exported function, but as executable file, because I want to call it as npm script. So this dynamic import is price for not creating another file which only allows to import truncate-db. May be its not really good justification